--- hostsblock/rc.conf	2012-12-09 18:28:46.000000000 +0000
+++ /etc/hostsblock/rc.conf	2013-04-17 01:21:36.027757802 +0000
@@ -101,12 +101,14 @@ blocklists=(
 ## HIGHLY RECOMMENDED LISTS
 'http://winhelp2002.mvps.org/hosts.zip'			# anti-ad and anti-malware list (winhelp2002.mvps.org/hosts.htm)
 'http://pgl.yoyo.org/as/serverlist.php?hostformat=hosts&mimetype=plaintext' # anti-ad (pgl.yoyo.org/adservers/)
-'http://hosts-file.net/download/hosts.zip'		# hpHosts main anti-ad and anti-malware (hosts-file.net/)
+#'http://hosts-file.net/download/hosts.zip'		# hpHosts main anti-ad and anti-malware (hosts-file.net/)
 'http://www.malwaredomainlist.com/hostslist/hosts.txt'  # anti-malware (www.malwaredomainlist.com/)
 'http://hosts-file.net/ad_servers.asp'			# anti-ads-only version of hpHosts main list (hosts-file.net)
+'http://sadi58.tripod.com/misc/hosts.txt'
 ## RECOMMENDED LISTS
-'http://hosts-file.net/hphosts-partial.asp'		# hpHosts inter-release (hosts-file.net/)
-'http://hostsfile.org/Downloads/BadHosts.unx.zip'	# anti-malware (hostsfile.org/hosts.html)
+#'http://support.it-mate.co.uk/downloads/HOSTS.txt' # default list
+#'http://hosts-file.net/hphosts-partial.asp'		# hpHosts inter-release (hosts-file.net/)
+#'http://hostsfile.org/Downloads/BadHosts.unx.zip'	# anti-malware (hostsfile.org/hosts.html)
 'http://hostsfile.mine.nu/Hosts.zip'			# anti-ad (hostsfile.mine.nu)
 'http://someonewhocares.org/hosts/hosts'		# anti-ad and anti-malware (someonewhocares.org/hosts/)
 'http://sysctl.org/cameleon/hosts'			# anti-ad (sysctl.org/cameleon/) 
